package.json

```diff
@@ -8,9 +8,9 @@
     "dev": "turbo dev --no-cache  --concurrency 16 --continue",
     "lint": "turbo lint",
     "prepare": "husky install",
-    "prettier-check": "prettier --check \"**/*.{js,ts,tsx,md,mdx}\"",
+    "prettier-check": "prettier --check \"**/*.{js,ts,tsx,md,mdx}\" \"!packages/svelte/**/*\" && pnpm -F @ai-sdk/svelte prettier-check",
```

Comment:
This is temporary; the plugins Svelte needs aren't compatible with Prettier 2.x, but the repo can pretty easily upgrade to 3.x -- just requires some formatting changes, which can be made in a subsequent PR (and then this can be changed back)

Reply:
do you want to take a stab at upgrading the repo separately? i was holding off bc we want to move to biome and replace eslint and prettier; however a prettier 3.0 upgrade in the meantime might maek sense